src/promptflow-core/promptflow/executor/_service/utils/batch_coordinator.py

```diff
@@ -83,7 +83,7 @@ async def exec_line(self, request: LineExecutionRequest):
     def exec_aggregation(self, request: AggregationRequest):
         """Execute aggregation nodes for the batch run."""
         with self._flow_executor._run_tracker.node_log_manager:
```

Comment:
You can remove this line because it is included in the `exec_aggregation`.

Reply:
Good catch, fixed.